Docs/Workflows.md

```diff
@@ -57,23 +57,24 @@ jobs:
       timeout-minutes: 10
 ```
 
-## Check Spelling workflow
+## Spelling workflow
 
 Checks for spelling mistakes in a repository using the [Check Spelling](https://github.com/marketplace/actions/check-spelling) GitHub Action. There are 3 configuration files for filtering false positives:
 - _`excludes.txt`_: This file includes file names and extensions to be ignored.
 - _`expect.txt`_: This file contains plain text words that would be considered a spelling mistake.
+- _`allow.txt`_: Same function as `expect.txt`.
```

Comment:
Why do we have a second way of doing the same thing?

Reply:
Added explanation.